ui/src/groups/NewGroup/NewGroupInvite.tsx

```diff
@@ -187,10 +195,20 @@ export default function NewGroupInvite({
         <button className="secondary-button" onClick={goToPrevStep}>
           Back
         </button>
-        <button className="button" onClick={goToNextStep}>
-          {shipsToInvite.length > 0
-            ? 'Invite People & Create Group'
-            : 'Create Group'}
+        <button className="button" onClick={goToNextStep} disabled={!ready}>
+          {ready ? (
+            submitText
+          ) : status === 'loading' ? (
+            <>
+              <LoadingSpinner className="mr-2 h-4 w-4" />
+              <span>Creating</span>
+            </>
+          ) : status === 'error' ? (
+            <>
+              <X16Icon className="mr-2 h-4 w-4" />
+              <span>Errored</span>
```

Comment:
Is it possible to show a more descriptive error to the user?

Reply:
The only thing we get back here is a stack trace so not really. Erroring here means the poke itself failed. I do wish we had more descriptive errors. I think this is an architectural/philosophical discussion that we need to have. Currently we just do lots of assertions, but it'd be nice to change those to not crash and actually send down useful errors.